docs/references/javascript/types/clerk-paginated-response.mdx

```diff
@@ -12,4 +12,4 @@ An interface that describes the response of a method that returns a paginated li
 | Properties | Type | Description |
 | --- | --- | --- |
 | `data` | `T[]` |  An array that contains the fetched data. |
-| `count` | `number` |  The total count of data that exist remotely. |
\ No newline at end of file
+| `total_count` | `number` |  The total count of data that exist remotely. |
```

Comment:
Let's check with the team about the return object having camelCase only

Reply:
let @BRKalow know internal chat 😄

Reply:
In JS it is `total_count`, in Backend it is `totalCount`